dot/rpc/modules/state.go

```diff
@@ -122,7 +121,7 @@ type StateGetReadProofResponse struct {
 // StorageChangeSetResponse is the struct that holds the block and changes
 type StorageChangeSetResponse struct {
 	Block   *common.Hash `json:"block"`
-	Changes [][]string   `json:"changes"`
+	Changes [][]*string  `json:"changes"`
```

Comment:
Could we the bottom slice as a sized-2 array?

```suggestion
	Changes [][2]*string  `json:"changes"`
```